### PR TITLE
Fix #6 - Provide ability to connect to db on non-default port

### LINKED ISSUE #6: Ability to connect to DB on non default ports
The DBConnect initializer assumes the port of the mysql db to be 3306 and does not provide any way to override this.

### PR DESCRIPTION
**Pull request by devashish2203**
Adding support to connect to non-default (default = 3306) port.
This change should be backward compatible.

### CODE REVIEW

**Comment by devashish2203** [comment]
@mastizada Please review..

File: dbConnect/dbConnect.py [hunk 1]
```diff
@@ -43,14 +44,16 @@ def connect(self):
...
         """
         Initialise object with credentials file provided
         """
         if host and user and password and database:
-            self.settings = {"host": host, "user": user, "password": password, "database": database}
+            self.settings = {"host": host, "user": user, "password": password, "database": database, "port": port}
         else:
             self.settings = json.load(open(credentials_file, 'r'))
+            if 'port' not in self.settings:
+                self.settings['port'] = 3306
```
Comment (mastizada) [inline dbConnect/dbConnect.py]: make it equal to port variable. We already initialised port parameter with default value 3306.
`self.settings['port'] = port`

Comment (devashish2203) [inline dbConnect/dbConnect.py]: done..

**Comment by mastizada** [comment]
Thanks @devashish2203 . Will release in new version this weekend

**Comment by devashish2203** [comment]
Glad to help.. :)

**Pull request merged**